src/components/FormSections/Delivery.jsx

```diff
@@ -13,12 +14,12 @@ function Delivery(props) {
 
             <p>
                 {content.mail_text}
-                <br />Division of Elections State of Alaska
-                <br />PO Box 110017
-                <br />Juneau, AK 99811-0017
+                <br />{stateAddress.office_name}
+                <br />{stateAddress.street_address}
+                <br />{stateAddress.city_state}
             </p>
 
-            <p>{content.rminder_text2}</p>
+            <p>{content.reminder_text2}</p>
```

Comment:
fixed spelling on this and added the string needed on the page... can reference the last screen shot to see